modules/ROOT/pages/apicat-publish-using-api-catalog-cli.adoc

```diff
@@ -0,0 +1,208 @@
+
+This example shows the `api-catalog publish-asset` command with both of these options. This example assumes that authentication credentials are set in environment variables.
+
+----
+> api-catalog publish-asset -a https://anypoint.mulesoft.com --descriptor-file=mydescriptor.yaml -o 1234abc-b34a-7cd3-4s32-12345abc2345 --force-publish 
+
+----
+
+[[trigger-conditions]]
+== Trigger the Publish Using Conditions 
+
+Custom trigger conditions enable you to control whether APIs are published when the API Catalog CLI command runs. This example shows how `--trigger-criteria` options correlate with descriptor file trigger conditions to control whether or not to publish the described APIs.   
+
+*Example Trigger Conditions in a Descriptor file*
+
+Following is an example `triggerConditions` section in the `mydescriptor.yaml` descriptor file at the global level:
+
+---
+ 
+ triggerConditions:
+  ref:
+   - main
+  tags:
+   - migrate
+   - republish
+  status:
+   - complete
+   - ready
+
+---
+
+*Example That Triggers the Publish*
+
+The following command triggers the API publish because the trigger option values match all of the global trigger conditions in the `mydescriptor.yaml` descriptor file. 
+
+----
+> api-catalog publish-asset -a https://anypoint.mulesoft.com -d mydescriptor.yaml -o 1234abc-b34a-7cd3-4s32-12345abc2345 -u myAnyPtAccount -p myPwd@4!myacct --trigger-criteria=ref:main --trigger-criteria=tags:migrate --trigger-criteria=status:complete
+
+----
+
+*Example That Does Not Trigger the Publish*
+
+The following command does not trigger the API publish because the trigger option value for `ref` is `test`, not `main`. Because not all trigger values match the corresponding global trigger conditions, no APIs are published. 
+
+----
+> api-catalog publish-asset -a https://anypoint.mulesoft.com -d mydescriptor.yaml -o 1234abc-b34a-7cd3-4s32-12345abc2345 -u myAnyPtAccount -p myPwd@4!myacct --trigger-criteria=ref:test --trigger-criteria=tags:migrate --trigger-criteria=status:complete
+
+----
+
+[[versionStrategyConditions]]
+== Set Version Strategy Using Conditions 
+
+Version strategy conditions enable you to set the version of assets when the API Catalog CLI publish command runs as part of your CI/CD pipeline or scripts. This example shows how `--version-strategy-criteria` options correlate with descriptor file version strategy conditions to set the version strategy for the described APIs. If the version strategy is set at the global level, it overrides values set at any other level. If the value for a condition matches more than one version strategy, the command gives an error.
+
+*Example Version Strategy Conditions in a Descriptor file*
+
+Following is an example `versionStrategyConditions` section in the `mydescriptor.yaml` descriptor file. 
+
+----
+ 
+versionStrategyConditions:
+patchIncrease:
+branch:
+ ‘main’
+author:
+ ‘admin’
+snapshot:
+branch:
+‘test’
+‘Develop’
+
+----
+
+*Example That Sets the Version Strategy During the Publish*
```

Comment:
```suggestion
*Example that sets the version strategy during the publish*
```

Reply:
So could this actually be "...the version strategy when you run the `publish` command."?